src/components/chronometer/chronometer.e2e.ts

```diff
@@ -0,0 +1,48 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+
+describe("gx-chronometer", () => {
```

Comment:
I'd like to have some tests where the chronometer is started/stopped/reset by changing the state property, instead of calling the methods.